docs/Basics/Working with ROIs.md

```diff
@@ -28,27 +28,45 @@ In the options parameter,`getRois()` has a `kind` option which tells what kind o
 
 :::
 
-<!-- Add  a comment about what the image is -->
```

Comment:
What I meant here is to give a little context about the the image we are working is. It's a microscopic image of some material, you can give the magnification factor and the problem at hand: what we try to extract from this image and why